cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,15 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('https://react-redux.realworld.io');
+    cy.contains('Sign up').click();
+    cy.get('input[placeholder="Username"]').type(user.username);
+    cy.get('input[placeholder="Email"]').type(user.email);
+    cy.get('input[placeholder="Password"]').type(user.password);
+    cy.get('button[type="submit"]').click();
+    cy.get('.navbar').should('contain', user.username);
   });
```

Comment:
It's a good practice to clean up after your tests to avoid any side effects that might affect other tests. In this case, you might want to consider deleting the user after the test finishes. You can do this in an afterEach or after hook.